bootstrap-app/src/cmr/bootstrap/api/routes.clj

```diff
@@ -30,45 +30,55 @@
   [params]
   (= "true" (when (:synchronous params) (str/lower-case (:synchronous params)))))
 
+(defn- get-dispatcher
```

Comment:
Ultimately, it might make sense to move this into the same place you define your protocols/core API. It's not a route-specific function, but instead a general "get me the implementation appropriate for the system configuration" ...